zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceMigrationPreconditions.java

```diff
@@ -57,7 +57,8 @@ public final class ProcessInstanceMigrationPreconditions {
           BpmnElementType.BUSINESS_RULE_TASK,
           BpmnElementType.SCRIPT_TASK,
           BpmnElementType.SEND_TASK,
-          BpmnElementType.MULTI_INSTANCE_BODY);
+          BpmnElementType.MULTI_INSTANCE_BODY,
+          BpmnElementType.PARALLEL_GATEWAY);
```

Comment:
💭 Initially, this change worried me as I thought it opens up the possibility to migrate process instances with already taken incoming sequence flows of joining parallel gateways. However, that's still blocked by the concurrent command check:

>Expected to migrate process instance '2251799813685253' but a concurrent command was executed on the process instance. Please retry the migration.

Reply:
Yes, it is. I also checked that validation logic beforehand.